### PR TITLE
Sets up docker environment

### PR DESCRIPTION
**Pull request by alaxalves**
Hey @IgorHalfeld I've created a sample docker environment for your app. But I'm having some trouble, first of all I'm not being able to run the app neither locally or using docker. Maybe there are some missing packages, or some hitfixes you've made only locally. Either way I'm not being able to reach the app at `8080`. Could you give me some hints? :D 
Related to #3 and #4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:10-slim
```
Comment (alaxalves) [inline Dockerfile]: Is this node version correct?

Comment (IgorHalfeld) [reply Dockerfile]: Hmm, `8.14.0-alpine` is better, can you change?

**Review by IgorHalfeld — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "cors": "^2.8.5",
     "express": "^4.16.4",
     "npm-run-all": "^4.1.5",
-    "vue": "^2.5.18"
+    "vue": "^2.5.18",
+    "vue-cli": "^2.9.6"
```
[the remark this answers is not in the document]
Comment (alaxalves) [reply package.json]: Just adding this didn't fix the issue #4 :/ and If I remove the vue-cli dependencie it breaks the app as:
``` shell
vue-redtube | > @ vue /vue-redtube
vue-redtube | > vue serve
vue-redtube | 
vue-redtube | sh: vue: not found
vue-redtube | npm ERR! file sh
vue-redtube | npm ERR! code ELIFECYCLE
vue-redtube | npm ERR! errno ENOENT
vue-redtube | npm ERR! syscall spawn
vue-redtube | npm ERR! @ vue: `vue serve`
vue-redtube | npm ERR! spawn ENOENT
vue-redtube | npm ERR! 
vue-redtube | npm ERR! Failed at the @ vue script.
vue-redtube | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
vue-redtube | 
vue-redtube | > @ proxy /vue-redtube
vue-redtube | > node proxy
vue-redtube | 
vue-redtube | 
vue-redtube | npm ERR! A complete log of this run can be found in:
vue-redtube | npm ERR!     /root/.npm/_logs/2018-12-09T01_43_03_091Z-debug.log
vue-redtube | ERROR: "vue" exited with 1.
vue-redtube | npm ERR! code ELIFECYCLE
vue-redtube | npm ERR! errno 1
vue-redtube | npm ERR! @ start: `$(npm bin)/npm-run-all --parallel proxy vue`
vue-redtube | npm ERR! Exit status 1
vue-redtube | npm ERR! 
vue-redtube | npm ERR! Failed at the @ start script.
vue-redtube | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
vue-redtube | 
vue-redtube | npm ERR! A complete log of this run can be found in:
vue-redtube | npm ERR!     /root/.npm/_logs/2018-12-09T01_43_03_588Z-debug.log
vue-redtube exited with code 1

```

File: Dockerfile (hunk 1 above)
Comment (tiagorosendo) [reply Dockerfile]: @IgorHalfeld Docker Aline have some troubles with dns resolution when you try to run the container inside a Kubernetes Cluster so I think that slim image will be a better option.

https://github.com/gliderlabs/docker-alpine/issues/255

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (IgorHalfeld) [reply package.json]: Hmm,  I install `@vue/cli` globally too and It works xD
check out the `Dockerfile`:

```dockerfile
FROM node:8.14.0-alpine

RUN apk add --update build-base && \
  npm install @vue/cli \
  npm install -g @vue/cli-service-global

ADD . /vue-redtube

WORKDIR /vue-redtube

ENTRYPOINT ["sh", "scripts/start.sh"]
```

[the remark this answers is not in the document]
Comment (IgorHalfeld) [reply package.json]: When you run with this `Dockerfile` above, It'll show this error: 

image

But this has already been fixed the master o/

[the remark this answers is not in the document]
Comment (IgorHalfeld) [reply package.json]: @alaxalves can you fix this <3?

[the remark this answers is not in the document]
Comment (alaxalves) [reply package.json]: Will do

File: README.md [hunk 3]
```diff
@@ -19,7 +19,23 @@
...
+$ docker run -it -p 8080:8080 vue-redtube vue serve
+```
+
+If you'd rather use Docker Compose, just go
+
+``` sh
+$ docker-compose up
+```
+
+Or if you like it raw, ( ͡° ͜ʖ ͡°), just go
```
Comment (IgorHalfeld) [inline README.md]: hahahahahha pretty cool!

**Review by IgorHalfeld — APPROVED** [review]

**Comment by IgorHalfeld** [comment]
Merged!

**Pull request merged**